client/me/notification-settings/settings-form/stream.jsx

```diff
@@ -65,7 +65,7 @@ export default React.createClass( {
 					}
 
 					return ( <StreamHeader stream={ this.props.stream } /> );
-				}() }
+				} )() }
```

Comment:
>Nice catch on these - were they breaking tests?

If I recall correctly, the code was failing to compile on the newer version of Babel.